pkg/scheduler/plugins/coscheduling/coscheduling.go

```diff
@@ -95,7 +96,10 @@ func New(obj runtime.Object, handle framework.Handle) (framework.Plugin, error)
 	} else {
 		controllerWorkers = int(*args.ControllerWorkers)
 	}
-	go podGroupController.Run(controllerWorkers, ctx.Done())
+	if extendedHandle, ok := handle.(frameworkext.ExtendedHandle); ok {
```

Comment:
if trans fail, waitingForLeader=nil;  podGroupController.Run should be executed anyway.

Reply:
fixed